app/src/main/java/com/github/dedis/student20_pop/ui/event/MeetingEventCreationFragment.java

```diff
@@ -0,0 +1,126 @@
+package com.github.dedis.student20_pop.ui.event;
+
+import android.content.Context;
+import android.content.Intent;
+import android.os.Bundle;
+import android.text.Editable;
+import android.text.TextWatcher;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Button;
+import android.widget.EditText;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.fragment.app.FragmentManager;
+
+import com.github.dedis.student20_pop.R;
+import com.github.dedis.student20_pop.model.Event;
+import com.github.dedis.student20_pop.model.MeetingEvent;
+import com.github.dedis.student20_pop.utility.ui.organizer.OnEventCreatedListener;
+
+import java.text.DateFormat;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Calendar;
+import java.util.Locale;
+
+public final class MeetingEventCreationFragment extends AbstractEventCreationFragment {
+
+    public static final String TAG = MeetingEventCreationFragment.class.getSimpleName();
+    public static final DateFormat TIME_FORMAT = new SimpleDateFormat("HH:mm", Locale.FRENCH);
+    public static final DateFormat DATE_FORMAT = new SimpleDateFormat("dd/MM/yyyy", Locale.FRENCH);
+
+    private EditText meetingTitleEditText;
+    private EditText meetingLocationEditText;
+    private EditText meetingDescriptionEditText;
+
+    private Button confirmButton;
+    private Button cancelButton;
+
+    private OnEventCreatedListener eventCreatedListener;
+
+    @Override
+    public void onAttach(Context context) {
+        super.onAttach(context);
+        try {
+            eventCreatedListener = (OnEventCreatedListener) context;
+        } catch (ClassCastException e) {
+            throw new ClassCastException(context.toString() + " must implement OnEventCreatedListener");
+        }
+    }
```

Comment:
No need for a try/catch, use 
```
if (context instanceof OnEventCreatedListener)
    eventCreatedListener = (OnEventCreatedListener) context;
else
    eventCreatedListener = (OnEventCreatedListener) context;
```

Reply:
Mmmmh not sure to get the point of doing that. Could you explain a bit ?

Reply:
Yes my brain failed the copy/paste. What I mean is that using exception when you can use a condition is a bad habit.

Reply:
Okay now I understand what you mean and I totally agree with it. I'll change that